extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -408,6 +424,18 @@ export class VisibilityManagerForDoc extends VisibilityManager {
   }
 
   /** @override */
+  getRootLayoutBox() {
+    if (getMode(this.ampdoc.win).runtime == 'inabox') {
+      const root = this.ampdoc.getRootNode();
+      const rootElement = dev().assertElement(
+          root.documentElement || root.body || root);
+      return this.viewport_.getLayoutRect(rootElement);
+    }
+    const size = this.viewport_.getSize();
+    return layoutRectLtwh(0, 0, size.width, size.height);
```

Comment:
might be my short-sight, but i really feel root size should be document size not the viewport size...